blocks/identity-block/features/account-management/_children/SocialEditableFieldContainer.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { PrimaryFont } from '@wpmedia/shared-styles';
+import styled from 'styled-components';
+import getThemeStyle from 'fusion:themes';
+import { useFusionContext } from 'fusion:context';
+import './social-editable-field-styles.scss';
+
+const StyledContainer = styled.div`
+  border-color: ${(props) => props.borderColor};
```

Comment:
don't think we have a better way of getting border-color per the mocks than this

Reply:
The mocks are not to be followed 100% - the button's can follow the out of the box social styles as they do on login/sign up pages

Reply:
oh, huh ok

Reply:
removed this border